client-js/app/elements/labrad-plot.ts

```diff
@@ -135,6 +149,11 @@ export class Plot extends polymer.Base {
       area: HTMLElement, totWidth: number, totHeight: number): void {
     const p = this;
 
+    // Make room for the color bar if necessary.
+    if (p.numIndeps == 2) {
+      p.margin.right += COLOR_BAR_WIDGET_SIZE;
```

Comment:
This changes the margin every time `createPlot` is called, which can happen multiple times. The margin gets larger and larger over time and the plot smaller and smaller, which is a fun effect but probably not what we want :-)